File: Day_14/Q56.cpp
#include <bits/stdc++.h>
using namespace std;
// Input The Array
void input_array(int arr[], int n)
{
    for (int i = 0; i < n; i++)
    {
        cin >> arr[i];
    }
}
// Finding Duplicate Elements
void duplicates(int arr[], int n)
{
    for (int i = 0; i <= n - 1; i++)
    {
        for (int j = i + 1; j <= n - 1; j++)
        {
            if (arr[j] == arr[i])
            {
                cout << arr[j] << "\t";
                break;
            }
        }
    }
}

int main()
{
    int n;
    cin >> n;
    // Input Validation
    if (n <= 0)
    {
        cout << "Invalid Input";
        return 0;
    }
    int arr[n];
    // Performing Required Actions By Calling The Functions
    input_array(arr, n);
    duplicates(arr, n);
    return 0;
}

File: Day_14/Q56.java
import java.util.*;
class Q56{
    //Input The Array
    static void input_array(int arr[],int n,Scanner sc){
        for(int i = 0;i < n;i++){
            arr[i] = sc.nextInt();
        }   
    }
    //Finding Duplicate Elements
    static void duplicate_elements(int arr[],int n){
        for(int i = 0;i <= n-1;i++){
            for(int j = i+1;j <= n-1;j++){
                if(arr[j] == arr[i]){
                    System.out.print(arr[j] + "\t");
                    break;
                }
            }
        }
    }
    public static void main(String[] args){
        Scanner sc = new Scanner(System.in);
        int n = sc.nextInt();
        //Input Validation
        if(n <= 0){
            System.out.print("Invalid Input");
        }
        //Performing Required Actions By Calling The Functions
        else{
             int arr[] = new int[n];
             input_array(arr,n,sc);
             duplicate_elements(arr, n);
        }
        sc.close();
    }
}